admin/acceptance.py

```diff
@@ -345,25 +433,12 @@ def start_cluster(self, reactor):
             ])
             commands = commands.on(success=lambda _: zfs_commands)
 
-        certificates_path = FilePath(mkdtemp())
```

Comment:
I don't think this refactor is great; FLOC-1932 is going to be removing configuration from vagrant runner, so that'll just need to reintroduce the code back here.